tests/Http/Event/ImportEventRequestHandlerTest.php

```diff
@@ -4392,6 +4392,104 @@ public function it_throws_if_priceInfo_has_no_main_language(): void
         $this->assertValidationErrors($event, $expectedErrors);
     }
 
+    /**
+     * @test
+     */
+    public function it_throws_if_priceInfo_has_duplicate_names(): void
+    {
+        $event = [
+            'mainLanguage' => 'nl',
+            'name' => [
+                'nl' => 'Pannenkoeken voor het goede doel',
+            ],
+            'terms' => [
+                [
+                    'id' => '1.50.0.0.0',
+                ],
+            ],
+            'location' => [
+                '@id' => 'https://io.uitdatabank.dev/places/5cf42d51-3a4f-46f0-a8af-1cf672be8c84',
+            ],
+            'calendarType' => 'permanent',
+            'priceInfo' => [
+                [
+                    'category' => 'base',
```

Comment:
Maybe the base category should have another index to make sure that case is also handled?